File: runners/go/payloads/cmd.go
package payloads

import (
	"os/exec"
	"strings"
)

type Cmd struct{}

func (c Cmd) Run(payload_data []byte) {
	payload_args := strings.Split(string(payload_data), " ")

	cmd := exec.Command(payload_args[0], payload_args[1:]...)
	cmd.Run()
}

File: runners/csharp/csharp/Payloads/cmd.cs
using System.Diagnostics;

namespace Runner.Payloads
{
    class Cmd : IPayload
    {
        void IPayload.Run(byte[] payload_data)
        {
            string args = "/c " + System.Text.Encoding.UTF8.GetString(payload_data, 0, payload_data.Length);
            Process.Start("cmd.exe", args);
        }
    }
}
